### ISSUE TITLE
Invalid ACM Entires

### ISSUE THREAD

**Issue by dmcconachie** [UM-ARM-Lab/kuka_iiwa_interface#107]
`victor.srdf` has some invalid ACM entries. For example

```
457     <disable_collisions link1="victor_left_gripper_fingerC_dist" link2="victor_right_gripper_fingerA_dist" reason="Never" />
458     <disable_collisions link1="victor_left_gripper_fingerC_dist" link2="victor_right_gripper_fingerB_dist" reason="Never" />
459     <disable_collisions link1="victor_left_gripper_fingerC_dist" link2="victor_right_gripper_fingerB_med" reason="Never" />
```

**Comment by dmcconachie** [comment]
I'm fixing this on the `peter_3d_rope` branch right now, will submit a PR for `master` later.

**Comment by PeterMitrano** [comment]
Fixed in #127
